packages/api-clients/open-api/delegationApi.yml

```diff
@@ -487,37 +487,40 @@ paths:
         required: true
         description: Payload for delegation rejection
       responses:
-        headers:
-          X-Metadata-Version:
-            $ref: "#/components/headers/MetadataVersionHeader"
         "200":
           description: Delegation rejected
+          headers:
+            X-Metadata-Version:
+              $ref: "#/components/headers/MetadataVersionHeader"
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Delegation"
         "400":
           description: Bad Request
+          headers:
+            X-Metadata-Version:
+              $ref: "#/components/headers/MetadataVersionHeader"
```

Comment:
Please double-check there are no more `X-Metadata-Version` headers in error responses